subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -89,17 +89,19 @@ static inline bool isr_rx_ci_adva_check(uint8_t tx_addr, uint8_t *addr,
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 #define PAYLOAD_BASED_FRAG_COUNT \
-	ceiling_fraction(CONFIG_BT_CTLR_ADV_DATA_LEN_MAX, \
-			 PDU_AC_PAYLOAD_SIZE_MAX)
-#define PAYLOAD_FRAG_COUNT MAX(PAYLOAD_BASED_FRAG_COUNT, BT_CTLR_DF_PER_ADV_CTE_NUM_MAX)
+		ceiling_fraction(CONFIG_BT_CTLR_ADV_DATA_LEN_MAX, \
+				 PDU_AC_PAYLOAD_SIZE_MAX)
+#define PAYLOAD_FRAG_COUNT \
+		MAX(PAYLOAD_BASED_FRAG_COUNT, BT_CTLR_DF_PER_ADV_CTE_NUM_MAX)
```

Comment:
Did anything change here?

Reply:
No functional change, but including changes (by redundant indentation) in this commit to contain these lines as related to other changes in this PR.